tests/components/watchyourlan/conftest.py

```diff
@@ -0,0 +1,74 @@
+"""Module containing fixtures for testing the WatchYourLAN integration."""
+
+from collections.abc import Generator
+from unittest.mock import AsyncMock, patch
+
+import pytest
+
+from homeassistant.components.watchyourlan.const import DOMAIN
+from homeassistant.const import CONF_URL, CONF_VERIFY_SSL
+from homeassistant.core import HomeAssistant
+
+from tests.common import MockConfigEntry
+
+
+@pytest.fixture
+def mock_setup_entry() -> Generator[AsyncMock]:
+    """Override async_setup_entry."""
+    with patch(
+        "homeassistant.components.watchyourlan.async_setup_entry", return_value=True
+    ) as mock_setup_entry:
+        yield mock_setup_entry
+
+
+@pytest.fixture
+async def mock_watchyourlan_client_fixture() -> AsyncMock:
+    """Fixture to mock WatchYourLANClient."""
+    with patch(
+        "homeassistant.components.watchyourlan.config_flow.WatchYourLANClient",
+    ) as mock_client:
+        instance = mock_client.return_value
+        instance.get_all_hosts = AsyncMock(return_value={})
+        yield instance
```

Comment:
Why don't you use this fixture as well in test_sensor.py if you also add the other patch?

Reply:
Check airgradient for an example